test/integration/testMaxLobSize.js

```diff
@@ -0,0 +1,240 @@
+const assert = require('assert');
+const async = require('async');
+const testUtil = require('./testUtil');
+const crypto = require('crypto');
+const snowflake = require('./../../lib/snowflake');
+const connOption = require('./connectionOptions');
+
+function generateRandomString(sizeInBytes) {
+  const bufferSize = Math.ceil(sizeInBytes / 2);
+  const buffer = crypto.randomBytes(bufferSize);
+  return buffer.toString('hex').slice(0, sizeInBytes);
+}
+
+describe('Max LOB test', function () {
+  let connection;
+  // This size cannot be tested on our env. The snowflake team should test internally.
+  // const MAX_LOB_SIZE = 128 * 1024 * 1024; 
+  const MAX_LOB_SIZE = 16 * 1024 * 1024;
+  const LARGE_SIZE = MAX_LOB_SIZE / 2;
+  const MEDIEUM_SIZE = LARGE_SIZE / 2;
+  const ORIGIN_SIZE = MEDIEUM_SIZE / 2; 
+  const SMALL_SIZE = 16;
+
+  const tableName = 'my_lob_test';
+  const createTable = `create or replace table ${tableName} (c1 varchar, c2 varchar, c3 int)`;
```

Comment:
I'm not sure whether the max lob changes would change the default size of varchar as well or need to set the size explicitly like varchar(128M). That would be an easy fix though we could confirm with Snowflake or wait for the test result.

Reply:
@sfc-gh-ext-simba-hx Thank you for your question. In my opinion after setting  "Max Lob size flags" the default size will be automatically changed to 128M. It works in tests.